### DISCUSSION TITLE
BleachBit 4.4.2

### DISCUSSION THREAD

**Discussion by Jeff220121** [bleachbit/bleachbit#1337] [Q&A]
👋 Hi, I’m @Jeff220121, anyone out there have any red-font error messages in BleachBit 4.4.2?

**Comment by abitrolly** [comment]
Hi @Jeff220121. Wanna paste some screens to brag about them? )

**Comment by PavanLuca** [comment]
![Schermata da 2024-03-14 18-21-09](https://github.com/bleachbit/bleachbit/assets/123581658/4207038c-01ac-4d57-9a35-09c1999c929a)
